.github/workflows/check-certificates.yml

```diff
@@ -0,0 +1,99 @@
+name: Check for issues with signing certificates
+
+on:
+  schedule:
+    # run every 10 hours
+    - cron: "0 */10 * * *"
+  # workflow_dispatch event allows the workflow to be triggered manually.
+  # This could be used to run an immediate check after updating certificate secrets.
+  # See: https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+
+env:
+  # Begin notifications when there are less than this many days remaining before expiration
+  EXPIRATION_WARNING_PERIOD: 30
+
+jobs:
+  get-certificates-list:
+    # This workflow would fail in forks that don't have the certificate secrets defined
+    if: github.repository == 'arduino/FirmwareUpdater'
+    runs-on: ubuntu-latest
+    outputs:
+      certificates: ${{ steps.get-files.outputs.certificates }}
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Set certificates path environment variable
+        run: |
+          # See: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable
+          echo "FILES=\\\"$(ls -md ${{ github.workspace }}/certs/* | xargs | sed 's/, /","/g')\\\"" >> $GITHUB_ENV
```

Comment:
The quoting is still not working right. I think it originates here.

In the debug output from when I ran it in my fork:
https://github.com/per1234/FirmwareUpdater/runs/2163628151?check_suite_focus=true#step:3:36
```
FILES='\"/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/AddTrustExternalCA_Root.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/BaltimoreCyberTrust_Root.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/Digicert_Root.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/EquifaxSecureCA_Root.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GeoTrustGlobalCA_Root.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GeoTrustPrimaryCertificationAuthority-G3.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GoDaddyRootCertificateAuthority-G2.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/SampleMatrixCA_Root.cer","/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/VeriSignClass3PublicPrimaryCertificationAuthority-G5.cer\"'
```
the first and last double quotes are escaped.

The JSON is not being correctly generated from this text:
https://github.com/per1234/FirmwareUpdater/runs/2163628151?check_suite_focus=true#step:4:39
```
::set-output name=certificates::{"cert_file":["/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/AddTrustExternalCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/BaltimoreCyberTrust_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/Digicert_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/EquifaxSecureCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GeoTrustGlobalCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GeoTrustPrimaryCertificationAuthority-G3.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GoDaddyRootCertificateAuthority-G2.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/SampleMatrixCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/VeriSignClass3PublicPrimaryCertificationAuthority-G5.cer"]}
```
which results in there only being a single `check-certificates` job which feeds the entire list of paths to `openssl`:
https://github.com/per1234/FirmwareUpdater/runs/2163629329?check_suite_focus=true#step:4:136
```
Can't open /home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/AddTrustExternalCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/BaltimoreCyberTrust_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/Digicert_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/EquifaxSecureCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GeoTrustGlobalCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GeoTrustPrimaryCertificationAuthority-G3.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/GoDaddyRootCertificateAuthority-G2.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/SampleMatrixCA_Root.cer,/home/runner/work/FirmwareUpdater/FirmwareUpdater/certs/VeriSignClass3PublicPrimaryCertificationAuthority-G5.cer for reading, No such file or directory
```

Reply:
I think I got it right now, I did some tests and it seems to work as expected.